### PR TITLE
Updates to mctheme after changes to fuelux repeater.

### LINKED ISSUE #328: mctheme updates for new repeater features
After updating the repeater with actions column and multi select need to update styling for mctheme

### PR DESCRIPTION
**Pull request by BenjaminNeilDavis**
Fixes #328

### CODE REVIEW

File: examples/fuelux/fuelux.html [hunk 1]
```diff
@@ -6,10 +6,13 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
 	<link href="../../bower_components/bootstrap/dist/css/bootstrap.css" rel="stylesheet" type="text/css">
-	<link href="../../bower_components/fuelux/dist/css/fuelux.css" rel="stylesheet" type="text/css">
+<!--	<link href="../../bower_components/fuelux/dist/css/fuelux.css" rel="stylesheet" type="text/css">-->
+
+	<link href="http://localhost:8000/dist/css/fuelux.css" rel="stylesheet" type="text/css">
```
Comment (chriscorwin) [inline examples/fuelux/fuelux.html]: Uh oh!

Must fix this before it can be merged, @BenjaminNeilDavis

**Comment by swilliamset** [comment]
fixed a issue found while reviewing. issued PR to @BenjaminNeilDavis's branch. after accepted @chriscorwin can begin review.

**Comment by BenjaminNeilDavis** [comment]
@chriscorwin I have merged @swilliamset changes so you can begin your review! Thanks!

**Comment by BenjaminNeilDavis** [comment]
On click of multi select item fix padding

image

**Comment by chriscorwin** [comment]
Looks fantastic. :+1:

**Pull request merged**